examples/dynamic-bridge-app/linux/include/Attribute.h

```diff
@@ -0,0 +1,80 @@
+#pragma once
```

Comment:
To better organize this stuff, can we please create a 'data-model' sub-folder which contains the *.cpp and *.h files for the common, library bits, and a separate 'generated' folder that contains all the clusters and device type bits?